### ISSUE TITLE
áđa

### ISSUE THREAD

**Issue by hoangnguyen15** [yixia/VitamioBundle#428]
how to play stream https://livecdn.fptplay.net/liver/vtv4_hls.smil/playlist.m3u8?token=eyJzZXJ2ZXJfdGltZSI6IDE0NzY1MDk1NjUsICJyZXNvdXJjZV9pZCI6ICJ2dHY0X2hscyIsICJwYXJ0bmVyX2lkIjogIjU3MTNiZThjMDdlNDJlZDRiOTE2Mzg4ZSIsICJ0b2tlbiI6ICJmMmYzMGNmN2VkNWFlMWM1NjRiMmFlYTUzMWU0N2RmMSIsICJ2YWxpZF9taW51dGVzIjogMTQ3NjU5NTk2NSwgInJtaXAiOiAiMS41NS4xMzQuMTQzIn0=
log error https protocol not found, recompile FFmpeg with openssl, gnutls or securetransport enabled. please support
Thank you

**Comment by MythodeaLoL** [comment]
HTTPS/SSL/OpenSSL is disabled, because **vitamio is outdated**.

**Comment by hoangnguyen15** [comment]
Thanks for the reply. how play stream https: //. please support. 
thank you

**Comment by appyo** [comment]
Try version 5.0.2. It supports ssl again.

http://wscdn.miaopai.com/download/VitamioBundle-5.0.2.zip

**Comment by hoangnguyen15** [comment]
yeah.. verry good. please. how to play stream : http://htvlive.1c656bad.cdnviet.com/htvc_movies.720p.stream/playlist.m3u8?t=yej14VVT+gy9j4cP3KWhr7eLmbVb4mvIgUFsfKoQYkWTln1kiDbh63PwtT8qUMlFnQnm54d0fJmf7Sm2WE6uGc4G4fUD51lb6wSm1uSi5bfz6LGvu1JqDiFptm2BtK40 
Vitamio play error :  avformat_open_input: Invalid data found when processing input : -1094995529
has text relocations. This is wasting memory and prevents security hardening. Please fix.

**Comment by appyo** [comment]
Is this message from emulator or from a real device? I have this message only from emulator or x86 device.  On armv7a device it works for me. Otherwise, there is a workaround if you set targetsdk to 22, then this message disappears. Your stream has a time based token, so i can't test it anymore.
